clusterloader2/testing/load/config.yaml

```diff
@@ -104,6 +114,9 @@ tuningSets:
     # as each RS changes its size from X to a uniform random value in [X/2, 3X/2].
     # To match 10 [pods/s] requirement, we need to divide saturationTime by 4.
     timeLimit: {{DivideInt $saturationTime 4}}s
+- name: DeletionTimeLimited
+  TimeLimitedLoad:
+    timeLimit: 5m
```

Comment:
I think you added it accidentally.  
Also, at the end we probably want to remove this 5m sleep. Could you put sleep within ifs?